.github/workflows/ci.yml

```diff
@@ -100,3 +100,27 @@ jobs:
       with:
         name: bin-${{ matrix.base-image }}
         path: bin/splunk-*.zip
+
+  create-release:
+    if: github.ref_type == 'tag'
+    name: Create GitHub release
+    runs-on: ubuntu-20.04
+    needs: [ build, container-build ]
+    permissions:
+      contents: write
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v3.3.0
+      - uses: actions/download-artifact@v3.0.2
+        with:
+          path: .
+      - name: Install zip
+        uses: montudor/action-zip@v1.0.0
+      - run: cd bin-alpine ; zip -qq -r ../splunk-opentelemetry-dotnet-linux-glibc.zip . * ; cd ..
+      - run: cd bin-ubuntu-20.04 ; zip -qq -r ../splunk-opentelemetry-dotnet-linux-musl.zip . * ; cd ..
+      - run: cd bin-windows-2022 ; zip -qq -r ../opentelemetry-dotnet-instrumentation-windows.zip . * ; cd ..
```

Comment:
The name should be splunk-opentelemetry-... I think

Reply:
#141